### PR TITLE
Set user agent in all Stackdriver requests

### PR DESCRIPTION
**Pull request by jkohen**
Can you advice on what tests and doc updates you'd like for this change?

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: plugins/outputs/stackdriver/stackdriver.go [hunk 1]
```diff
@@ -65,6 +67,7 @@ var sampleConfig = `
   #   namespace = "myapp"
   #   location = "eu-north0"
 `
+var userAgent = fmt.Sprintf("telegraf %s", internal.Version())
```
Comment (danielnelson) [inline plugins/outputs/stackdriver/stackdriver.go]: Can you use `internal.ProductToken()`?

**Comment by danielnelson** [comment]
Thanks for the pr, I think manual testing will be fine on this one.

File: plugins/outputs/stackdriver/stackdriver.go (hunk 1 above)
Comment (jkohen) [reply plugins/outputs/stackdriver/stackdriver.go]: Good tip, done.

**Comment by jkohen** [comment]
This is ready for your review.

**Pull request merged**